frontend/app/src/components/MainMenu/MainMenu.test.tsx

```diff
@@ -152,31 +147,32 @@ describe("MainMenu", () => {
     })
   })
 
-  it("should not render set of configurable elements", () => {
+  it("should not render set of configurable elements", async () => {
     const menuItems = {
       hideGetHelp: true,
       hideReportABug: true,
       aboutSectionMd: "",
     }
     const props = getProps({ menuItems })
-    const wrapper = mount(<MainMenu {...props} />)
-    const popoverContent = wrapper.find("StatefulPopover").prop("content")
-    // @ts-expect-error
-    const menuWrapper = mount(popoverContent(() => {}))
-
-    // @ts-expect-error
-    const menuLabels = menuWrapper
-      .find("MenuStatefulContainer")
-      .at(0)
-      .prop("items")
-      .map(item => item.label)
-    expect(menuLabels).toEqual([
+    render(<MainMenu {...props} />)
+    await openMenu(screen)
+
+    // first SubMenu (menu items, not dev menu items)
+    const coreMenu = screen.getAllByTestId("main-menu-list")[0]
```

Comment:
I know there probably wasn't any tests for the dev menu but should we add a check for the dev menu?

Reply:
There are existing checks covering dev menu options in the test file 👍🏼